app/src/main/java/ar/com/wolox/android/example/ui/session/SessionActivity.java

```diff
@@ -1,19 +1,33 @@
 package ar.com.wolox.android.example.ui.session;
 
+import javax.inject.Inject;
+
 import ar.com.wolox.android.R;
+import ar.com.wolox.android.example.ui.home.HomeFragment;
+import ar.com.wolox.android.example.ui.login.LoginFragment;
+import ar.com.wolox.android.example.utils.UserSession;
 import ar.com.wolox.wolmo.core.activity.WolmoActivity;
+import kotlin.text.StringsKt;
 
 /**
  *  Session activity. It will be in charge of managing the logged user's info.
  */
 public class SessionActivity extends WolmoActivity {
+
+    @Inject
+    public UserSession userSession;
```

Comment:
Is there a better way to do this? Dagger doesn't let me inject a private field 😢